docs/account_access.md

```diff
@@ -1,22 +1,24 @@
-# account\_access
+# account_access
 
 [<-Back-to:Auth](database-auth.md)
 
-**The \`account\_access\` table**
+**The `account_access` table**
 
 This table holds security access level for any realm in [realmlist](realmlist) table.
 
 **Structure**
 
-| Field        | Type       | Attributes | Key | Null | Default | Extra | Comment |
-|--------------|------------|------------|-----|------|---------|-------|---------|
-| [id][1]      | int(10)    | unsigned   | PRI | NO   |         |       |         |
-| [gmlevel][2] | tinyint(3) | unsigned   |     | NO   |         |       |         |
-| [RealmID][3] | int(11)    | signed     | PRI | NO   | -1      |       |         |
+| Field        | Type         | Attributes | Key | Null | Default | Extra | Comment |
+|--------------|--------------|------------|-----|------|---------|-------|---------|
+| [id][1]      | int(10)      | unsigned   | PRI | NO   | NULL    |       |         |
+| [gmlevel][2] | tinyint(3)   | unsigned   |     | NO   | NULL    |       |         |
```

Comment:
They dont have a default value so, no, the default is not NULL
![image](https://user-images.githubusercontent.com/24550914/104133424-c7a17300-5383-11eb-8a67-c10459f3ced1.png)

Reply:
also if it doesnt allow NULL, then it can never be NULL =)

Reply:
> They dont have a default value so, no, the default is not NULL
> ![image](https://user-images.githubusercontent.com/24550914/104133424-c7a17300-5383-11eb-8a67-c10459f3ced1.png)

The problem is that it is in the structure of the table in that way, so leave the SQL at the bottom, because if not, it means that we have to modify the SQL.